docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: Working with Etsy in Gatsby
+---
+
+## Etsy in Gatsby
+
+This guide will walk you through the steps needed in order to successfully work with Etsy e-commerce sites in Gatsby.
+
+## Prerequisites
+
+- Create a new [Etsy account](https://www.etsy.com/join) and store if you don’t have one.
+- Create a private app in your store by navigating to Apps, then Manage private apps.
```

Comment:
I wasn't able to find this in the UI. I would direct people to "[Register as a developer](https://www.etsy.com/developers/register)".